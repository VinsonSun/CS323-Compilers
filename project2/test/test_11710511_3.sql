int test3() {
   int x = 1.0;
   float y = 1.1;
   return y;
}
