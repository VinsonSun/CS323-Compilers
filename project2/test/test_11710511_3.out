Error type 5 at Line 2: unmatching type on both sides of assignment
Error type 8 at Line 4: incompatiable return type
